### DISCUSSION TITLE
Show Privacy Policy when the website is visited by someone the first time

### DISCUSSION THREAD

**Discussion by ObiKeahloa** [monkeytypegame/monkeytype#997] [Ideas] [answered]
The title says everything most websites do this just a idea might be better if done , not a big problem to me but might be to some (As I myself only found the privacy policy after checking through the source code.

**Comment by Miodec** [comment, marked as answer]
The privacy policy is always available in the footer as it is for most websites on the internet. I dont think showing it automatically is necessary.
